packages/web/app/edge/routing/route.ts

```
@@ -189,18 +189,19 @@ function getUserContext(request: NextRequest): UserContext {
 
   const config = ROUTING_CONFIGS[tier]
 
+  const freeConfig = ROUTING_CONFIGS.free!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)